README.md

```diff
@@ -53,11 +63,40 @@ message Cell {
   }
 }
 ```
-See the `model` subproject for details.
+Definitely, we can use this model but it is so easy to make some mistake when 
+the model becomes invalid state. What can we do to avoid this?
+
+It would be much better if we could add the following:
+
+1. Mark some fields as `required` to admit that this is important
+to initialize these fields.
+2. Set the minimum acceptable value for the `side_size` field.
+3. Add some validation rules that verify the size of the `cell` collection.
```

Comment:
"Add some validation rules that verify the size of the `cell` collection. Indeed, a tic-tac-toe board should be square, and hence have an appropriate number of cells, being the size of the side squared."